modules/pytket-honeywell/pytket/extensions/honeywell/backends/honeywell.py

```diff
@@ -346,6 +350,11 @@ def process_circuits(
             body["name"] = circ.name if circ.name else f"{self._label}_{i}"
             body["program"] = honeywell_circ
             body["count"] = n_shots
+            if final_index > 0:
+                body["batch-exec"] = batch_exec
+                if i == final_index:
+                    body["batch-end"] = True
+            print(body)
```

Comment:
This looks like a debugging statement that was left in - do we actually want to output `body`?